cms/urls.py

```diff
@@ -92,9 +92,9 @@
     url(r'^checklists/{}/(?P<checklist_index>\d+)?$'.format(settings.COURSE_KEY_PATTERN), 'checklists_handler'),
     url(r'^orphan/{}$'.format(settings.COURSE_KEY_PATTERN), 'orphan_handler'),
     url(r'^assets/{}/{}?$'.format(settings.COURSE_KEY_PATTERN, settings.ASSET_KEY_PATTERN), 'assets_handler'),
-    url(r'^import/{}$'.format(settings.COURSE_KEY_PATTERN), 'import_handler'),
+    url(r'^import/{}$'.format(settings.COURSE_KEY_PATTERN), 'course_import_handler'),
     url(r'^import_status/{}/(?P<filename>.+)$'.format(settings.COURSE_KEY_PATTERN), 'import_status_handler'),
```

Comment:
I think you should just change this to use `COURSELIKE_KEY_PATTERN` (defined above in this file), and then you can get rid of the redundant `library_import_status_handler` method and URL altogether.

Wherever possible, we should try to find places to cut down on the `if course: course_fn() else library_fn()`.